SessionMessagingKit/Open Groups/blocklist.txt

```diff
@@ -0,0 +1 @@
+fd65dd2b888a492b6e5dd2d064ab8b01871c06f67d7bdea3775b30d445f5f4a0
```

Comment:
Looks like this already needs to be updated 😅

Reply:
Yeah, there is another hash for test and I didn't include that into this PR...